global.yaml

```diff
@@ -402,3 +402,8 @@ global:
   # This field is set to false by default, so 'helm template ...'
   # will ignore the helm test yaml files when generating the template
   enableHelmTest: false
+
+# If set, no iptable init will be added. It assumes CNI is installed.
+# TODO: rename to 'enableIptables' or add 'interceptionMode: CNI'
+istio_cni:
```

Comment:
This should be under values.yaml - global should only have the global settings.

Reply:
This is how it works in Istio 1.1. I agree it should be changed as long as we are ok with the backwards incompatibility

Reply:
I mean: global.yaml file should have only the global settings - there is a top-level global: 

We want to keep 1.1 compat - if 1.1 used --set istio_cni.enabled=true, it should keep working.
That means anything using istio_cni (injector ?) should have that in its values.yaml

Reply:
That makes sense, thanks